### PR TITLE
Base setup

### PR DESCRIPTION
**Pull request by iharmanpannu**
### **Added**

- Eleventy for templates and site generation.
- Rollup for straightforward JS asset bundling.
- Alpine JS A rugged, minimal framework for composing JavaScript behavior in markup.
- Tailwindcss for a utility first CSS workflow.
- PurgeCSS for optimizing CSS output.

### CODE REVIEW

**Comment by iharmanpannu** [comment]
> LGTM. At some point in the future (probably after the V3 launch) we may want to investigate moving to Rollup instead of Webpack, but I'm fine with going with it for now.  Thanks Harmann!

Yea I agree I prefer rollup too will definitely look into it

**Review by JudahGabriel — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,12 @@
 {
-  "editor.formatOnSave": true
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (jgw96) [inline .vscode/settings.json:2]: Not sure if we need to check in our vscode editor settings? @iharmanpannu Am I missing something here ?

**Review by jgw96 — CHANGES_REQUESTED** [review]

File: src/scripts/copy.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  if (preTags !== null) {
+    for (let i = 0; i < preTags.length; i++) {
+      if (!isPrismClass(preTags[i])) continue
+      preTags[
+        i
+      ].innerHTML = `<div class="copy">copy</div><code class="${preTags[i].className}">${preTags[i].innerHTML}</code>`
+    }
+  }
+
+  const clipboard = new ClipboardJS(".copy", {
```
[the remark this answers is not in the document]
Comment (iharmanpannu) [reply src/scripts/copy.js:18]: Got it thanks for the feedback 👍🏼

File: .vscode/settings.json (hunk 1 above)
Comment (iharmanpannu) [reply .vscode/settings.json]: Fixed

**Pull request merged**